tests/unit_tests/new/Commands/Env/ViewCommandTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Pantheon\Terminus\UnitTests\Commands\Env;
+
+use Pantheon\Terminus\Commands\Env\ViewCommand;
+
+class ViewCommandTest extends EnvCommandTest
+{
+    public function testView()
+    {
+        $this->env->expects($this->any())
+            ->method('domain')
+            ->willReturn('dev-my-site.example.com');
+
+        $command = new ViewCommand();
+        $command->setSites($this->sites);
+        $url = $command->view('my-site.dev', ['print' => true]);
+        $this->assertEquals('http://dev-my-site.example.com/', $url);
+    }
+
+    public function testViewLocked()
+    {
+        $this->env->expects($this->any())
+            ->method('domain')
+            ->willReturn('dev-my-site.example.com');
+        $this->env->expects($this->any())
+            ->method('get')
+            ->with('lock')
+            ->willReturn(
+                (object)[
+                    'locked' => true,
+                    'username' => 'user',
+                    'password' => 'pass',
+                ]
+            );
+
+        $command = new ViewCommand();
+        $command->setSites($this->sites);
```

Comment:
I think we should keep putting this step in the `setUp` function to keep things DRY.